examples/hello_world/src/lib.rs

```diff
@@ -31,27 +31,23 @@ use std::io::Write;
 #[derive(OakNode)]
 struct Node;
 
-impl Node {
-    fn say_hello(&self, req: &HelloRequest) -> HelloResponse {
-        let mut res = HelloResponse::new();
-        info!("Say hello to {}", req.greeting);
-        res.reply = format!("HELLO {}!", req.greeting);
-        res
-    }
+// TODO: Generate this code via a macro or code generation (e.g. a protoc plugin).
+trait HelloWorldNode {
+    fn say_hello(&self, req: &HelloRequest) -> HelloResponse;
 }
 
+// TODO: Generate this code via a macro or code generation (e.g. a protoc plugin).
 impl oak::Node for Node {
     fn new() -> Self {
         oak_log::init(log::Level::Debug).unwrap();
         Node
     }
     fn invoke(&mut self, grpc_method_name: &str, grpc_channel: &mut oak::Channel) {
         let mut logging_channel = oak::logging_channel();
-        // TODO: Generate this code via a macro or code generation (e.g. a protoc plugin).
         match grpc_method_name {
```

Comment:
Eventually we need to somehow get this into the generated trait, or generate a separate stand alone method that dispatches from grpc to the trait. Do you have any ideas about that already? :)